### PR TITLE
[chore] Accelerate process of marking issues as stale

### LINKED ISSUE #15684: [admin] mark-issues-as-stale.sh not grabbing enough issues
### Describe the issue you're reporting

The new workflow to mark issues as stale worked, but I don’t think it processed enough issues since the gh command didn’t set a limit. 

I think this call https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/92ad54f707a967386b764b6fdaea5a2ba4377319/.github/workflows/scripts/mark-issues-as-stale.sh#L40 needs to include a [—limit flag](https://cli.github.com/manual/gh_issue_list). At the moment I suspect it is using the default of 30.

### PR DESCRIPTION
**Pull request by evan-bradley**
**Description:**

This PR temporarily shortens the time between an issue being marked stale and closed to 60 days to expedite closing out the very old issues in the backlog. This should close out the oldest issues roughly by the start of next year. Issues that are determined to still be relevant after this period can be reopened. After the current backlog of stale issues is processed, we can restore the 120 day period between being marked stale and being closed.

This is a follow-up to https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15719, which will allow issue openers to easily add labels for issues without a component label with a CODEOWNERS file entry and notify the code owners if they think their issue is still relevant. Code owners can also add any labels they feel are relevant to their issue.

**Link to tracking Issue:**

Fixes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/15684

### CODE REVIEW

**Review by TylerHelmuth — APPROVED** [review]
@evan-bradley we cant merge this until https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/15719 is merged correct?

**Comment by evan-bradley** [comment]
We _can_, but I think we should wait. I'll mark it as a draft for now.

**Comment by evan-bradley** [comment]
@open-telemetry/collector-contrib-approvers @open-telemetry/collector-contrib-triagers please take a look.

**Review by Aneurysm9 — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**
